src/StatusQ/Components/StatusTagSelector.qml

```diff
@@ -113,6 +113,13 @@ Item {
     */
     property var compressedKeyGetter: (pubKey) => { /*return compressed key;*/ }
 
+    /*!
+        \qmlproperty var StatusTagSelector::colorIdForPubkey
+        This property holds the function to calculate the color Id
+        based on the public key.
+    */
+    property var colorIdForPubkey: (pubKey) => { /*return color Id;*/ }
```

Comment:
Should we stay consistent and suffix this with `Getter`? (I'd go for `Fn` but other functions here use `Getter`)

Reply:
done!